git/Cargo.toml

```diff
@@ -1,8 +1,10 @@
 [package]
 name = "git"
-version = "0.1.0"
+version = "0.0.1"
 authors = ["Intent HQ <engineering@intenthq.com>"]
-edition = "2018"
+edition = "2019"
+description = "Gitkv is a server for using git as a key value store for text files"
```

Comment:
This is not correct as this is the description of the whole project or the `gitkv-server`, no this library